docs/index.md

```diff
@@ -1,10 +1,130 @@
+
+Plan the Site
+-------------
+
+If you have not done so already, [plan the overall architecture of your OSG site](site-planning). It is recommended that your plan be sufficiently detailed to include the OSG hosts that are needed and the main software components for each host. Be sure to consider [the operating systems that OSG supports](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SupportedOperatingSystems). A simple way to organize this information is in a table; for example, a basic site might include:
+
+| Purpose              | Host                                | Major Software                                           |
+|:---------------------|:------------------------------------|:---------------------------------------------------------|
+| Compute Element (CE) | `osg-ce.example.edu`                | OSG CE, HTCondor Central Manager, etc. (`osg-ce-condor`) |
+| Worker Nodes         | `wNNN.cluster.example.edu`          | OSG worker node client (`osg-wn-client`)                 |
+
+Prepare the Batch System
+------------------------
+
+The assumption is that you have an existing batch system at your site. Currently, we support [HTCondor](http://research.cs.wisc.edu/htcondor/), [LSF](http://www-03.ibm.com/systems/platformcomputing/products/lsf/), [PBS](http://www.pbsworks.com) and [TORQUE](http://www.adaptivecomputing.com/products/open-source/torque/), [SGE](http://en.wikipedia.org/wiki/Oracle_Grid_Engine), and [Slurm](http://slurm.schedmd.com) batch systems.
+
+For smaller sites (less than 50 worker nodes), the most common way to add a site to OSG is to install the OSG Compute Element (CE) on the central host of your batch system.  At such a site - especially if you have minimal time to maintain a CE - you may want to contact goc@opensciencegrid.org to ask about using an OSG-hosted CE instead of running your own.  Before proceeding with an install, be sure that you can submit and successfully run a job from your OSG CE host into your batch system.
+
+Add OSG Software
+----------------
+
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+
+!!! note
+    For sites with more than a trivial number of hosts, it is recommended to use some sort of configuration management tool to install, configure, and maintain your site. While beyond the scope of OSG’s documentation to explain how to select and use such a system, some popular configuration management tools are [Puppet](http://puppetlabs.com), [Chef](https://www.chef.io), [Ansible](https://www.ansible.com), and [CFEngine](http://cfengine.com).
+
+### General Installation Instructions ###
```

Comment:
The rest of this document turns into a massive link dump, looking a lot like the nav bar (and referencing outdate stuff like OSG 3.1 and OSG 3.2).

Each sub-section should include a sentence or two on which things to read.  Given the emphasis on new admins in the first half of the document, we might want to skew toward new admins here too -- and downplay things like upgrading or esoteric topics for old sites (think: SHA-2 compliance).